### PR TITLE
integration: add test for getPublicKeys

### PR DESCRIPTION
**Pull request by keepkeyjon**

### CODE REVIEW

File: integration/src/bitcoin/bitcoin.ts [hunk 1]
```diff
@@ -61,6 +61,59 @@ export function bitcoinTests (get: () => { wallet: HDWallet, info: HDWalletInfo
       expect(await info.btcSupportsCoin('Testnet')).toBeTruthy()
     }, TIMEOUT)
 
+    test.only('getPublicKeys', async () => {
```
Comment (keepkeyjon) [inline integration/src/bitcoin/bitcoin.ts]: FIXME: remove the `.only`

File: integration/src/bitcoin/bitcoin.ts [hunk 2]
```diff
@@ -61,6 +61,59 @@ export function bitcoinTests (get: () => { wallet: HDWallet, info: HDWalletInfo
...
+        curve: 'secp256k1',
+        scriptType: BTCInputScriptType.SpendWitness
+      }, {
+        coin: 'Bitcoin',
+        addressNList: bip32ToAddressNList(`m/0'/0'/0'`),
+        curve: 'secp256k1',
+        scriptType: BTCInputScriptType.SpendAddress
+      }, {
+        coin: 'Litecoin',
+        addressNList: bip32ToAddressNList(`m/0'/0'/0'`),
```
Comment (keepkeyjon) [inline integration/src/bitcoin/bitcoin.ts]: FIXME: would also be nice to test paths like `m/`, `m/42'`, etc, but the current implementation in `hdwallet-ledger` assumes addressNList has length == 3.

Comment (millsmcilroy) [reply integration/src/bitcoin/bitcoin.ts]: Nice to have...

Comment (keepkeyjon) [reply integration/src/bitcoin/bitcoin.ts]: we fixed that bug though

**Pull request merged**